recipe/meta.yaml

```diff
@@ -32,34 +27,28 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
-    - libarrow                               # [build_platform != target_platform]
-    - pyarrow                                # [build_platform != target_platform]
```

Comment:
That's the pandas extra and not the base package. We can add that as an additional output but we should not pollute the base package with optional Requirements.

Reply:
It has always been an "optional" requirement that is baked into conda because conda doesn't have the same concept

Reply:
Optional requirements should be separate outputs. It has been a requirement in the past as we have devendored libarrow. This is not happening anymore as upstream doesn't vendor it anymore.

Reply:
what is this separate output that packages should list in their conda requirements if they require the `snowflake-connector-python[pandas]` dependencies?

Reply:
This needs to be created in the recipe.

Reply:
Right, so if my recipe currently contains `snowflake-connector-python` what do I need to update it to so that I get the `snowflake-connector-python[pandas]` dependencies?

Reply:
@xhochy Can you help?

Reply:
Okay, it looks like optional dependencies are handled by `run_constrained`, so if a specific version of pyarrow is required at some point in the future, it will be added there?